.github/workflows/backend.yml

```diff
@@ -131,9 +124,39 @@ jobs:
       - name: Running cluster test
         run: |
           /bin/bash ${{ matrix.case.script }}
+  schema-check-prepare:
+    name: schema-check-prepare-${{ matrix.version }}
+    needs: paths-filter
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [ "3.1.9", "3.2.0" ]
+    steps:
+      - uses: actions/cache/restore@v4
+        name: Download Binary Package Cache
+        with:
+          path: /tmp/${{ matrix.version }}
+          key: apache-dolphinscheduler-${{ matrix.version }}-bin
+          restore-keys: apache-dolphinscheduler-${{ matrix.version }}-bin
+      - name: Download Binary Package if not cached
+        run: |
+          if [ ! -f /tmp/${{ matrix.version }}/apache-dolphinscheduler-${{ matrix.version }}-bin.tar.gz ]; then
+            echo "Binary package not found in cache, downloading..."
+            mkdir -p /tmp/${{ matrix.version }}
+            wget https://mirrors.tuna.tsinghua.edu.cn/apache/dolphinscheduler/${{ matrix.version }}/apache-dolphinscheduler-${{ matrix.version }}-bin.tar.gz -P /tmp/${{ matrix.version }}
```

Comment:
Since `https://archive.apache.org` download speed is too slow and often causes timeout failures. So using `https://mirrors.tuna.tsinghua.edu.cn` to do the cache instead.

Notice: This site does not contains the archived versions.

Reply:
not sure use tsinghua mirrors is good idea or not

Reply:
if accept can we add some desc in L144?

Reply:
> not sure use tsinghua mirrors is good idea or not

After I tested a dozen mirrors, I found this source to be the fastest in github action.

> if accept can we add some desc in L144?

It seems comments don't seem to be supported in `run`. I tried, but I got an error...

Reply:
how about using apache cdn, such as `https://dlcdn.apache.org/dolphinscheduler/3.2.2/apache-dolphinscheduler-3.2.2-bin.tar.gz` for version 3.2.2?

Reply:
> It seems comments don't seem to be supported in run. I tried, but I got an error... 

I mean we and comment before the line for `run` property.

Reply:
> how about using apache cdn, such as `https://dlcdn.apache.org/dolphinscheduler/3.2.2/apache-dolphinscheduler-3.2.2-bin.tar.gz` for version 3.2.2?

And how about apache official cdn?

Reply:
> how about using apache cdn, such as `https://dlcdn.apache.org/dolphinscheduler/3.2.2/apache-dolphinscheduler-3.2.2-bin.tar.gz` for version 3.2.2?

This is the same speed as `This is the same speed as archive`...

Reply:
all right, so maybe we can keep tsinghua mirror here